reference-architecture/aws-ansible/playbooks/openshift-setup.yaml

```diff
@@ -25,12 +25,11 @@
     openshift_hosted_router_replicas: 3
     openshift_hosted_registry_replicas: 3
     openshift_master_cluster_method: native
-    openshift_node_local_quota_per_fsgroup: 512Mi
```

Comment:
You're right, if it is not defined the default value is none https://github.com/openshift/openshift-ansible/blob/81cf5123366814d098b7ca169a6e4deb2759507d/roles/openshift_node/tasks/main.yml#L33 , I'll try to fix it